java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -105,6 +105,12 @@
           "dictionary or not will be retained thereafter."),
   BLOOM_FILTER_COLUMNS("orc.bloom.filter.columns", "orc.bloom.filter.columns",
       "", "List of columns to create bloom filters for when writing."),
+  BLOOM_FILTER_WRITE_VERSION("orc.bloom.filter.write.version",
+      "orc.bloom.filter.write.version", OrcFile.BloomFilterVersion.UTF8.toString(),
+      "Which version of the bloom filter should we write."),
```

Comment:
I see from below to provide the option of writing both default and utf-8 we are using this config. Can you use enum of "default","utf-8" and add a description saying default will write both versions.